src/main/kotlin/com/example/mykku/dailymessage/domain/DailyMessageComment.kt

```
@@ -0,0 +1,31 @@
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Long? = null,
+
+    @Column(name = "content", nullable = false)
+    var content: String,
+
+    @Column(name = "like_count", nullable = false)
+    var likeCount: Int = 0,
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "daily_message_id")
+    val dailyMessage: DailyMessage,
```

detekt (1.23.8)

[warning] The class or object LikeFeed is empty.

(detekt.empty-blocks.EmptyClassBlock)